FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.m

```diff
@@ -121,6 +150,55 @@ - (FIRTrace *)activeTrace {
   return self.backgroundSessionTrace;
 }
 
++ (BOOL)isPrewarmAvailable {
+  if (![[GULAppEnvironmentUtil applePlatform] isEqualToString:@"ios"]) {
+    return NO;
+  }
+  NSString *systemVersion = [GULAppEnvironmentUtil systemVersion];
+  if ([systemVersion length] == 0) {
+    return NO;
+  }
+  NSArray *versionComponents = [systemVersion componentsSeparatedByString:@"."];
+  NSInteger majorVersion = [versionComponents[0] integerValue];
+  return majorVersion >= 15;
+}
+
+- (BOOL)isActivePrewarmEnabled {
+  return ([self.configurations prewarmDetectionMode] == 1 ||
```

Comment:
Can we replace these constants with something more understandable. For example: you can create a macro/enums for these values and we can use that check here. A check for the value of 1 or 3 is not so trivial to understand.